### ISSUE TITLE
obspy.taup.TauPyModel.get_travel_times() throws an exception for shallow depths

### ISSUE THREAD

**Issue by wmszeliga** [obspy/obspy#1214]
Using depths deeper than 0.0 km and shallower than 1.25 km with iasp91 model result in 

AttributeError: 'TauModel' object has no attribute 'ray_param'

Equivalent input values in java taup results in valid numbers.

```
import obspy.taup
model = obspy.taup.TauPyModel(model='iasp91')
arrivals = model.model.get_travel_times(distance_in_degree=0.9,source_depth_in_km=0.1)
```

reproduces the exception.  Changing the value of source_depth_in_km to 0.0, or 1.25 results the expected behavior.

**Comment by QuLogic** [comment]
Duplicate of #1057; this problem will be fixed in 0.11.0.

**Comment by wmszeliga** [comment]
Thanks, I searched for a related bug report, but couldn’t come up with the right phrase.

Cheers,
Walter
